src/typescript/azure-openapi-validator/rules/Rpaas_CreateOperationAsyncResponseValidation.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from "../rule"
+export const Rpaas_CreateOperationAsyncResponseValidation: string = "Rpaas_CreateOperationAsyncResponseValidation"
+
+rules.push({
+  id: "R4011",
```

Comment:
Please can you change the rule id,  this is a existing rule id  and make sure the rule id unique . And the R4014-R4019 have been reserved

Reply:
ah ok. sure.